src/hooks/useLocalStorage/index.ts

```diff
@@ -0,0 +1,53 @@
+import { defaultTo } from 'lodash'
+import { useState } from 'react'
+
+interface useLocalStorageHook {
+  setLocalValue: (value: string) => void
+  storedLocalValue: string
+  removeLocalItem: () => void
+}
+export const useLocalStorage = (
```

Comment:
Any particular reason to use a custom implementation instead of an existing package like https://github.com/nas5w/use-local-storage#readme ?